docker/hapi-compose.yml

```diff
@@ -28,7 +28,9 @@ services:
       - hapi-fhir-db:/var/lib/postgresql/data
     
   hapi-server:
-    image: "hapiproject/hapi:latest"
+    # Using an older version as the latest image broke our build.
+    # image: "hapiproject/hapi:latest"
+    image: "hapiproject/hapi:v6.1.0"
```

Comment:
@omarismail94 FYI, I had to use an older version of HAPI as the latest docker image (which was released today) breaks our build (I have not investigated why yet).